qutebrowser/browser/downloads.py

```diff
@@ -710,7 +713,14 @@ def _after_create_parent_question(self,
             last_used_directory = os.path.dirname(self._filename)
 
         log.downloads.debug("Setting filename to {}".format(self._filename))
-        if force_overwrite:
+        conflicting_downloads = self._get_conflicting_downloads()
+        if conflicting_downloads:
+            txt = "<b>{}</b> is already downloading. " \
```

Comment:
nit: The "already exists and is a special file" uses parenthesis to handle a line break instead of a backslash which is a little more idiomatic, at least for this codebase.